eslint/eslintrc-shared.yml

```diff
@@ -15,38 +18,136 @@ extends:
 rules:
     # 2 means error, 1 means warning, 0 means ignore.
 
+    # Four-space indent.
+    # Note: Some problems with const.
+    # indent: ["error", 4]
+    no-mixed-spaces-and-tabs: "error"
+    no-trailing-spaces: "error"
+
+    # Newlint at end of file.
+    eol-last: "error"
+
+    # Line comments start with a space <-- like this one does.
+    spaced-comment: "error"
+
+    # var compositeThing = new CompositeThing();
+    # var params = { thisIsPreferred: true };
+    # var params = { this_is_okay: true };
+    camelcase: ["error", { "properties": "never" }]
+    new-cap: "error"
+
+    # if (cond) {
+    #     stuff();
+    # } else {
+    #     otherStuff();
+    # }
+    # for (var i = 0; i < target; ++i) {
+    #     stuff();
+    # };
+    # var foo = function (arg1, arg2) {
+    #     stuff(arg1);
+    # };
+    # function factorial (value) {
+    #     return value == 0 ? 1 : value * factorial(value - 1);
+    # };
+    keyword-spacing: ["error", { "before": true, "after": true }]
+    brace-style: ["error", "1tbs", { "allowSingleLine": true }]
+    space-before-blocks: ["error", "always"]
+    curly: "error"
+    semi-spacing: "error"
+    func-style: "error"
+    space-before-function-paren: ["error", { "anonymous": "always", "named": "always" }]
+    comma-spacing: "error"
+    func-names: ["error", "never"]
+
+    # var foo = [
+    #     'one',
+    #     'two',
+    #     'three',
+    # ];
+    # var foo = ['one', 'two', 'three'];
+    # var bar = {
+    #     one: 1,
+    #     two: 2,
+    #     three: 3,
+    # };
+    # var bar = { one: 1, two: 2, three: 3 };
+    # var baz = { bazinga: {
+    #    one: 1,
+    #    two: 2,
+    # } };
+    # foo.bar = 123;
+    comma-style: ["error"]
```

Comment:
So, this doesn't cover the `require-trailing-comma` case, right? I'm personally fine with that, but I wanted to make sure I wasn't missing something.